### DISCUSSION TITLE
why NDK r17c can compile successfully and r25b cannot?

### DISCUSSION THREAD

**Discussion by xxxxxxLD** [android/ndk#1980] [Q&A]
I build the code by NDK-build -B with NDK r17c, it works successfully, but it doesn't work in NDK r25b with the same code and android.mk files. The error shows that undefined symbol: std::terminate() and undefined symbol : operator delete[](void *) , seems like some basic functions are undefined. Is this the bug of NDK r25b? The same question still appears in NDK-r26b.

**Comment by pirama-arumuga-nainar** [comment]
It's not a known issue.  We can help look if you share a reproducer.
